Crash after log out

Reproducible on an emulator running Android 8.1.

**To reproduce:**

1. Install the app
2. Log in using an email address
4. After a successful login, go to Settings and log out
5. Notice the crash
6. The app can't be restarted anymore without clearing the data

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.automattic.simplenote.debug/com.automattic.simplenote.NotesActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at com.automattic.simplenote.models.Account.hasVerifiedEmail(Account.java:43)
        at com.automattic.simplenote.Simplenote.checkReviewAccountOrVerifyEmail(Simplenote.java:237)
        at com.automattic.simplenote.Simplenote.access$800(Simplenote.java:58)
        at com.automattic.simplenote.Simplenote$ApplicationLifecycleMonitor.onActivityCreated(Simplenote.java:381)
        at android.app.Application.dispatchActivityCreated(Application.java:197)
        at android.app.Activity.onCreate(Activity.java:1015)
        at androidx.core.app.ComponentActivity.onCreate(ComponentActivity.java:81)
        at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:149)
        at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:313)
        at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:106)
        at com.automattic.simplenote.ThemedAppCompatActivity.onCreate(ThemedAppCompatActivity.java:25)
        at com.automattic.simplenote.NotesActivity.onCreate(NotesActivity.java:191)
        at android.app.Activity.performCreate(Activity.java:7009)
        at android.app.Activity.performCreate(Activity.java:7000)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6494) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
```

--- a/Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java
+++ b/Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java
@@ -227,8 +227,8 @@ public class Simplenote extends Application implements HeartbeatListener {
             return;
         }
 
-        if (mSimperium == null || mSimperium.getUser() == null) {
-            // Show nothing when Simperium or user cannot be retrieved.
+        if (mSimperium == null || mSimperium.getUser() == null || mSimperium.getUser().getEmail() == null) {
+            // Show nothing when Simperium, user, or email cannot be retrieved.
             return;
         }
 

